PhotoBooth/Camera Page/CameraViewControllerViewModel.swift

```diff
@@ -80,6 +82,29 @@ class CameraViewControllerViewModel: CameraViewControllerViewModeling {
         }
     }
 
+    func checkCameraAccess() -> Bool {
+        switch AVCaptureDevice.authorizationStatus(for: .video) {
+        case .denied:
+            print("Denied, request permission from settings")
```

Comment:
why are these print statements still here?

Reply:
you can combine these cases into 
case .denied, .restricted, .notDetermined: false
case .authorized: return true

you need to do a self review